save-running-config-action-pack/save-running-config/script.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the COPYING file.
+
+# This script just saves the running config of a device
+
+# Why?
+# If the running config were to change due to patch but this was still compliant with the 
+# designed config, CVP will not notice a change and the config will need saving.
```

Comment:
Not necessary, but might be nice to expand a little on why it needs saving, though that might just be me, as I'm unfamiliar with the some of the specifics of the running config and why this would be the case

Reply:
Tbh there might be a few reasons why the running config will need to be saved. I'll make this more generic.